bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -47,24 +47,40 @@ spec:
             source "/env/${K8S_NODE}"
             set +o allexport
           fi
-          echo "$(date -Iseconds) - starting ovs-daemons"
-          chown -R openvswitch:openvswitch /run/openvswitch
-          chown -R openvswitch:openvswitch /etc/openvswitch
-          function quit {
-              # Don't allow ovs-vswitchd to clear datapath flows on exit
-              kill -9 $(cat /var/run/openvswitch/ovs-vswitchd.pid 2>/dev/null) 2>/dev/null || true
-              kill $(cat /var/run/openvswitch/ovsdb-server.pid 2>/dev/null) 2>/dev/null || true
-              exit 0
-          }
-          trap quit SIGTERM
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --system-id=random
-          ovs-appctl vlog/set "file:${OVS_LOG_LEVEL}"
-          /usr/share/openvswitch/scripts/ovs-ctl --protocol=udp --dport=6081 enable-protocol
-          echo "$(date -Iseconds) - ovs-daemons running"
+          export SYSTEMD_IGNORE_CHROOT=yes
+          # Check to see if ovs is provided by the node:
+          if systemctl status ovsdb-server; then
```

Comment:
make this check if ovs is enabled in systemd, not check if it is actually up

Reply:
add another check if it is not up in this if block, then we need to exit until MCO/systemd restarts OVS

Reply:
done